tempesta_fw/connection.c

```diff
@@ -119,6 +120,7 @@ tfw_connection_new(struct sock *sk, int type, void *handler,
 
 	return 0;
 }
+EXPORT_SYMBOL(tfw_connection_new);
```

Comment:
It seems lb/ should not be designed as standalone modules, so the function should not be exported. See the below for about tfw_lb_srv_connect().